http3/server.go

```diff
@@ -308,11 +308,11 @@ func (s *Server) serveListener(ln quic.EarlyListener) error {
 		if err != nil {
 			return err
 		}
-		go func() {
+		go func(conn quic.EarlyConnection) {
```

Comment:
because it's a in a for loop we need to pass the conn as a value to the go-routine to avoid race conditions

Reply:
That's not correct. It only applies when you pass in the loop variable.